packages/node/src/serverless-functions/helpers.ts

```diff
@@ -1,21 +1,36 @@
-import type {
-  VercelRequest,
-  VercelResponse,
-  VercelRequestCookies,
-  VercelRequestQuery,
-  VercelRequestBody,
-} from './types';
-import { Server } from 'http';
-import type { Bridge } from './bridge';
-
-function getBodyParser(req: VercelRequest, body: Buffer) {
+import type { ServerResponse, IncomingMessage } from 'http';
+import { serializeBody } from '../utils';
+import { PassThrough } from 'stream';
+
+type VercelRequestCookies = { [key: string]: string };
+type VercelRequestQuery = { [key: string]: string | string[] };
+type VercelRequestBody = any;
+
+export type VercelRequest = IncomingMessage & {
+  query: VercelRequestQuery;
+  cookies: VercelRequestCookies;
+  body: VercelRequestBody;
+};
+
+export type VercelResponse = ServerResponse & {
+  send: (body: any) => VercelResponse;
+  json: (jsonBody: any) => VercelResponse;
+  status: (statusCode: number) => VercelResponse;
+  redirect: (statusOrUrl: string | number, url?: string) => VercelResponse;
+};
+
+class ApiError extends Error {
```

Comment:
_Ugh_ we have an APIError already in `packages/cli`. I think this is fine as is, but makes me think about how we'd want to hoist some of these error interfaces.

Reply:
you're right, I can see `APIError` and `responseError` at `packages/cli`, but I'm how I can import one of them here.

Note this class was ported from https://github.com/vercel/vercel/blob/main/packages/node-bridge/helpers.ts#L228